compiler/ballerina-parser/src/test/resources/misc/ambiguity/ambiguity_source_14.bal

```diff
@@ -1,5 +1,5 @@
 function foo() {
     function (int a) f1;
 
-    function(int a) => 5 -> f1;
+    function(int a) => 5->f1;
```

Comment:
any reason for these changes? now both `ambiguity_source_14` and `ambiguity_source_17.bal` has diagnostics.